ui/components/Link.tsx

```diff
@@ -3,6 +3,14 @@ import { Link as RouterLink } from "react-router-dom";
 import styled from "styled-components";
 import Text, { TextProps } from "./Text";
 
+function isAllowed(uri: string): boolean {
+  // Regex from https://github.com/cure53/DOMPurify/blob/cce00ac40d33c2aae6422eaa59e6a8aad5c73901/src/regexp.js
+  const regex = new RegExp(
+    /^(?:(?:(?:f|ht)tps?|mailto|tel|callto|cid|xmpp):|[^a-z]|[a-z+.\-]+(?:[^a-z+.\-:]|$))/i // eslint-disable-line no-useless-escape
+  );
+  return regex.test(uri);
+}
```

Comment:
Could we export this and add to index.ts?

Reply:
I shouldn't need you to prompt to do this, should I.